### PR TITLE
nmv: prefer selection if -u is active

### PR DESCRIPTION
**Pull request by N-R-K**
the rename plugin always asks for "selection vs current" even when -u flag is active. pass it to the plugin via `NNN_PREFER_SELECTION` so that there's less distracting prompts.

### CODE REVIEW

**Comment by N-R-K** [comment]
This also looks like something a lot of other plugin can benefit from as well. Should we export it for all plugins ?

**Comment by jarun** [comment]
Yes, please export for all plugins.

File: plugins/.nmv [hunk 1]
```diff
@@ -38,12 +39,17 @@ exit_status=0
 dst_file=$(mktemp "$TMPDIR/.nnnXXXXXX")
 
 if [ -s "$selection" ]; then
-	printf "Rename 'c'urrent / 's'election? "
-	read -r resp
+	printf "%s\n" "$NNN_PREFER_SELECTION" >&2
```
Comment (jarun) [inline plugins/.nmv:42]: Why do we need this print?
Also, can we make this a common function in `.nnn-plugin-helper`?

**Review by jarun — CHANGES_REQUESTED** [review]

File: plugins/.nmv (hunk 1 above)
Comment (N-R-K) [reply plugins/.nmv:42]: > Why do we need this print?

Was a debug print. I'll remove. :upside_down_face: 

> Also, can we make this a common function in `.nnn-plugin-helper`?

I'll take a look, and see what other plugins could use this.

**Comment by jarun** [comment]
Thank you!

**Comment by jarun** [comment]
Please check the build break:

```
########## shellcheck ##########

In plugins/.nmv line 43:
	arr=$(tr '\0' '\n' < "$selection")
                              ^--------^ SC2154: selection is referenced but not assigned.


In plugins/.nnn-plugin-helper line 51:
        [ -n "$1" ] && printf "$1 "
                              ^---^ SC2059: Don't use variables in the printf format string. Use printf "..%s.." "$foo".


In plugins/gpge line 24:
		files=$(tr '\0' '\n' < "$selection")
                                        ^--------^ SC2154: selection is referenced but not assigned.

For more information:
  https://www.shellcheck.net/wiki/SC2154 -- selection is referenced but not a...
  https://www.shellcheck.net/wiki/SC2059 -- Don't use variables in the printf...

Exited with code exit status 1
```

**Pull request merged**